server_transport/src/wstcp.c

```diff
@@ -3321,7 +3321,7 @@ static char * http_escapepath(char * out, const char * in) {
     return ret;
 }
 
-const uint8_t keybase [] = {
+const unsigned char keybase [] = {
```

Comment:
uint8_t is a better type, more in keeping with what we do elsewhere. I think this (and all the other uint changes)  should be reverted. What motivated the change?

Reply:
openSSL3.0 exposed APIs to symmetric encryption/decryption are accepting unsigned char form of key and iv

`int EVP_EncryptInit_ex(EVP_CIPHER_CTX *ctx, const EVP_CIPHER *type,
                       ENGINE *impl, const unsigned char *key, const unsigned char *iv);`

I found, having key and iv defining in the same data type would be helpful here and discard unnecessary type conversation.

Please let me know, if we still requires key and iv in uint type.

Reply:
As discussed in call, used unsigned char in old and new implementation (ubi8 as well as ubi9)